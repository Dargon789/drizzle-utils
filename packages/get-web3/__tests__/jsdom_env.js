/**
 * @jest-environment jsdom
 */
const getWeb3 = require("../index");

describe("get-web3 tests in browser environment", () => {
  test("getWeb3 function exists", () => {
    expect(getWeb3).toBeDefined();
  });

  test("default fallback provider exists", async () => {
    const web3 = await getWeb3();
    expect(web3.currentProvider).toMatchSnapshot();
  });
});
