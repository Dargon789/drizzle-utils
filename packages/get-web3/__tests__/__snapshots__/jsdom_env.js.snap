// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`get-web3 tests in browser environment default fallback provider exists 1`] = `
HttpProvider {
  "connected": false,
  "headers": undefined,
  "host": "http://127.0.0.1:9545",
  "httpAgent": Agent {
    "_events": Object {
      "free": [Function],
    },
    "_eventsCount": 1,
    "_maxListeners": undefined,
    "defaultPort": 80,
    "freeSockets": Object {},
    "keepAlive": true,
    "keepAliveMsecs": 1000,
    "maxFreeSockets": 256,
    "maxSockets": Infinity,
    "options": Object {
      "keepAlive": true,
      "path": null,
    },
    "protocol": "http:",
    "requests": Object {},
    "sockets": Object {},
  },
  "timeout": 0,
}
`;
